mvvm-state/src/commonMain/kotlin/dev/icerock/moko/mvvm/ResourceState.kt

```diff
@@ -4,18 +4,18 @@
 
 package dev.icerock.moko.mvvm
 
-sealed class ResourceState<out T, out E> {
-    data class Success<out T, out E>(val data: T) : ResourceState<T, E>()
-    data class Failed<out T, out E>(val error: E) : ResourceState<T, E>()
-    class Loading<out T, out E> : ResourceState<T, E>()
-    class Empty<out T, out E> : ResourceState<T, E>()
+sealed class ResourceState<out TData, out TError> {
```

Comment:
this change should support simple automatically migration if we want to merge it...try to write some regexp which will allow us and others migrate to new version in minutes on any size of project codebase